src/main/java/homeworks/homework_4/Exersise_10.java

```diff
@@ -0,0 +1,23 @@
+package homeworks.homework_4;
+
+import java.util.Scanner;
+
+public class Exersise_10 {
+
+  public static void main(String[] args) {
+
+       int [] array = {12, 104, 81};
+        int accum = 0;
+
+        for(int i = 0; i < array.length; i++) {
```

Comment:
wrong implementation, try to debug next solution: 
        for(int i = 0; i < array.length; i++) {
            char[] chars = Integer.toString(array[i]).toCharArray();
            for(int k = 0; k < chars.length; k++) {
                accum = accum + Integer.parseInt(String.valueOf(chars[k]));
            }
        }